### ISSUE TITLE
Source maps getting loaded as script

### ISSUE THREAD

**Issue by eliot-akira** [jaredpalmer/after.js#326]
### Current Behavior

After creating a new project with `create-after-app`, I'm seeing errors due to `*.js.map` files getting loaded as scripts.

![image](https://user-images.githubusercontent.com/5352434/80311042-c1f80300-87dd-11ea-81e4-a6020f09938c.png)

![image](https://user-images.githubusercontent.com/5352434/80311094-ee138400-87dd-11ea-8cd8-0981bb10ee20.png)

In `src/server.js`, these source maps are included in `process.env.RAZZLE_CHUNKS_MANIFEST`.

As a quick test, I tried the following to remove them:

```js
const chunks = require(process.env.RAZZLE_CHUNKS_MANIFEST);

for (const key of Object.keys(chunks)) {
  if (chunks[key].js) {
    chunks[key].js = chunks[key].js.filter(f => f.slice(-4)!=='.map')    
  }
}
```

That solves it, but I figured this should be filtered out either in Razzle or After.js?

### Expected behavior

I'd expect source maps to not be loaded as script.

### Reproducible example

```sh
npx create-after-app test
cd test
yarn start
```

Visit `http://localhost:3000` and open dev console.

**Comment by nimaa77** [comment]
I will push a fix later today

**Comment by eliot-akira** [comment]
Sounds great! Thank you for looking into it.

**Comment by jwrlee88** [comment]
Hi @nimaa77 ! I am in the midst of upgrading my after.js project to version 3.0.1 and after successfully upgrading I get the same error in my chrome console:

[https://i.imgur.com/QvpPzLE.png](imgur link:)

Any update on when this will be fixed?

**Comment by nimaa77** [comment]
This issue is fixed in v3.0.2 

GG
